docs/src/domains/steps/dns.md

```diff
@@ -1,170 +1,139 @@
-
+To map incoming requests to the appropriate container,
+Platform.sh runs routers in [each region](../../development/regions.md).
+A router's IP address can change in two cases:
+- During an upgrade or maintenance operation, routers can be taken offline while changes are applied.
+- During a region upscale or downscale, routers can be added or removed.
+
+To prevent downtime in such situations,
+create `CNAME` records through your DNS provider.
+ 
+`CNAME` records map a domain to another.
+Set up your `CNAME` records to point at your project's [target edge hostname](../../domains/steps/_index.md#2-get-the-target-for-your-project).
+Each Platform.sh edge hostname comes with a DNS record
+that resolves the IP addresses of the routers in your project's region.
+When a router's IP address changes,
+it's then automatically and immediately updated.
+
+Note that `CNAME` records can't point to [apex domains](../../other/glossary.md#apex-domain).
+Check with your DNS provider if they offer [workarounds](#workarounds-for-apex-domains).
+
+## Workarounds for apex domains
+ 
+Although `CNAME` records can't point to apex domains, most DNS providers offer workarounds.
 The recommended approach is to use custom records.
-
+ 
+Contact your DNS provider and choose the option that works for you.
+ 
 {{< codetabs >}}
-
+ 
 +++
 title=Use custom records
 file=none
 highlight=false
 +++
-
-Some DNS providers offer custom, non-standard records (sometimes `ANAME` or `ALIAS` records) that you can manage like `CNAME` records.
-These nonstandard records make an internal lookup behind the scenes and respond to DNS lookups as if they were `A` records.
-As these are nonstandard, their behavior (and quality) can vary and not all DNS registrars offer such a feature.
-
-If you want your site to be accessible at a URL like `https://example.com` and not only `https://www.example.com`,
-this is the best way to do so.
-
-To configure your domain name to point to your project using custom records, follow the instructions on [how to set up a custom domain](./_index.md).
-When you come to configuring your DNS provider, replace the suggested `CNAME` record with the custom record pointing from your domain to the target.
-
-Examples of such workaround records and providers include:
-
-<!-- vale Platform.condescending = NO -->
-* `CNAME` flattening at [CloudFlare](https://developers.cloudflare.com/dns/additional-options/cname-flattening)
-* `ANAME` records at [easyDNS](https://easydns.com/features/aname-root-domain-alias/),
-  [DNS Made Easy](https://support.dnsmadeeasy.com/support/solutions/articles/47001001412-aname-records),
-  and [Name.com](https://www.name.com/support/articles/115010493967-adding-an-aname-alias-record)
-* `ALIAS` records at [DNSimple](https://support.dnsimple.com/articles/alias-record/)
-  and [ClouDNS](https://www.cloudns.net/wiki/article/18/)
-<!-- vale Platform.condescending = YES -->
-
```

Comment:
I went back and forth on whether I should keep all the links or not. Open to opinions.